crda/remotematcher.go

```diff
@@ -0,0 +1,404 @@
+package crda
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"golang.org/x/sync/errgroup"
+
+	"github.com/quay/zlog"
+	"go.opentelemetry.io/otel/baggage"
+	"go.opentelemetry.io/otel/label"
+
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/libvuln/driver"
+)
+
+var (
+	_ driver.Matcher             = (*Matcher)(nil)
+	_ driver.RemoteMatcher       = (*Matcher)(nil)
+	_ driver.MatcherConfigurable = (*Matcher)(nil)
+)
+
+const (
+	// Bounded concurrency limit.
+	defaultBatchSize          = 100
```

Comment:
Reduce the batchSize to 10, having more than 10 would cause higher latency. Ideally the backend shouldn't allow batchSize > 10

Reply:
ACK.